third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -537,13 +537,7 @@ function Pause_evaluateInFrame({ frameId, expression }) {
     return buildRrpObjectResult({ result: argsCdp });
   }
 
-  let rv = null;
-  try {
-    rv = doEvaluation();
-  }
-  catch (err) {
-    log(`[RuntimeError] evaluateInFrame err: ${err?.stack || err}`);
-  }
+  const rv = doEvaluation();
```

Comment:
@klochek:
This is a sneaky/tangential change. Sorry I should have put it in the description. Updated the description now:

* Removing try/catch around `evaluate` commands. I added those to prevent some command crashes I did not want to deal with at the time because they were crashing hard. Adding it back in now that we have proper command crash triage.